### PR TITLE
fix: update meeting name after fetch meetinginfo

### PR DESCRIPTION
**Pull request by rsarika**
Issue: Meeting name not showing in meeting widget instead it shows sip address.
Fix: added update title code in fetchMeetingInfo.topic

https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-566707

Screenshot 2024-10-09 at 7 26 00 PM
Screenshot 2024-10-09 at 7 26 13 PM
Screenshot 2024-10-09 at 7 27 58 PM

### CODE REVIEW

File: src/MeetingsSDKAdapter.js [hunk 1]
```diff
@@ -595,9 +595,11 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
       )),
       tap((meeting) => {
         const sdkMeeting = this.fetchMeeting(meeting.ID);
+        const updatedTitle = sdkMeeting.meetingInfo.topic || meeting.title;
+        const updatedMeeting = {...meeting, title: updatedTitle};
```
[the remark this answers is not in the document]
Comment (rsarika) [reply src/MeetingsSDKAdapter.js]: Addressed

File: src/MeetingsSDKAdapter.js [hunk 2]
```diff
@@ -595,9 +595,11 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
...
 
-        this.meetings[meeting.ID] = meeting;
-        sdkMeeting.emit(EVENT_MEETING_UPDATED, meeting);
+        this.meetings[meeting.ID] = updatedMeeting;
+        sdkMeeting.emit(EVENT_MEETING_UPDATED, updatedMeeting);
```
[the remark this answers is not in the document]
Comment (rsarika) [reply src/MeetingsSDKAdapter.js]: yes, attached screenshots

File: src/MeetingsSDKAdapter.js [hunk 3]
```diff
@@ -634,6 +637,13 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     try {
       const sdkMeeting = this.fetchMeeting(ID);
 
+      // eslint-disable-next-line no-console
+      console.log('ravi joinMeeting sdkMeeting ', sdkMeeting);
```
Comment (Kesari3008) [inline src/MeetingsSDKAdapter.js:641]: Please remove this console logs before merge

File: src/MeetingsSDKAdapter.js [hunk 4]
```diff
@@ -661,6 +671,10 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
               }),
             }));
         } else {
+          // update the meeting title if available after password verification
```
Comment (Kesari3008) [inline src/MeetingsSDKAdapter.js:674]: Why did we need this code and above now?

**Review by Kesari3008 — APPROVED** [review]
Test different options : Join meeting using Person ID, email and SIP URI and verify title works in all three

File: src/MeetingsSDKAdapter.js (hunk 4 above)
Comment (rsarika) [reply src/MeetingsSDKAdapter.js:667]: removed from above

File: src/MeetingsSDKAdapter.js (hunk 3 above)
Comment (rsarika) [reply src/MeetingsSDKAdapter.js]: removed

**Pull request merged**